client/Lykke.Job.OperationsCache.Client/AutorestClientMapper.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Lykke.Service.OperationsCache.Client
+﻿namespace Lykke.Job.OperationsCache.Client
```

Comment:
I'm against renaming Job to Service in this case, but because of change urgency have to approve